source/tests/system/nidmm_driver_api_tests.cpp

```diff
@@ -0,0 +1,282 @@
+#include <gtest/gtest.h>
+
+#include "nidmm/nidmm_library.h"
+#include "nidmm/nidmm_service.h"
+
+namespace ni {
+namespace tests {
+namespace system {
+
+namespace dmm = nidmm_grpc;
+
+const int kDMMDriverApiSuccess = 0;
+
+class NiDMMDriverApiTest : public ::testing::Test {
```

Comment:
`NiDMMDriverApiTest` --> `NiDmmDriverApiTest` for consistency with comments made in the metadata PR.

Reply:
changed